api/opencl_runtime_layer.asciidoc

```diff
@@ -8061,6 +8061,7 @@ include::{generated}/api/version-notes/clEnqueueNDRangeKernel.asciidoc[]
     values that describe the offset used to calculate the global ID of a
     work-item.
     If _global_work_offset_ is `NULL`, the global IDs start at offset (0, 0, 0).
+    _global_work_offset_ has to be `NULL` <<unified-spec, before>> version 1.1.
```

Comment:
Nice catch.  This looks good to me, but could I suggest rephrasing slightly to say "_global_work_offset_ **must** be `NULL`" rather than "**has to** be `NULL`", to be more consistent with terminology in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)?  I know we don't follow RFC 2119 consistently in the spec today, but I think this something we _should_ be doing, and if we use this phrasing here now it's one fewer place we'll need to fix in the future.

Thanks!

Reply:
Good point! I updated the specification text and the commit message.